node-proxy/lib/utils/http-utils.js

```diff
@@ -29,12 +29,12 @@ exports.createProtocolServer = function(protocol, opts) {
       break;
     case 'https':
       var ssl_opts  = {
-        secureProtocol: 'SSLv23_method',
         secureOptions: constants.SSL_OP_NO_SSLv3
       };
       ssl_opts.ca   = fs.readFileSync(opts.ca);
       ssl_opts.cert = fs.readFileSync(opts.certificate);
       ssl_opts.key  = fs.readFileSync(opts.private_key);
+      ssl_opts.ciphers = opts.ciphers;
```

Comment:
My biggest concern is with how we handle upgrades.  

...although I just checked and found that `/etc/openshift/web-proxy-config.json` is not marked `%config`(!), so that does not appear to be a problem.